packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.stories.js

```diff
@@ -62,6 +62,22 @@ export const Default = ({ dark }) => {
   )
 }
 
+export const MobileWidth = ({ dark }) => {
+  const themeMode = dark ? 'dark' : 'light'
+  return (
+    <Grommet background={background} theme={zooTheme} themeMode={themeMode}>
+      <Provider classifierStore={store}>
+        <Box width='449px'>
```

Comment:
Storybook includes predefined device viewports, which you can pass into stories as parameters. 
https://github.com/zooniverse/front-end-monorepo/blob/9a89c350990e0fb98cabd23ee97c955039036b00/packages/app-project/src/screens/ProjectHomePage/components/Hero/Hero.stories.js#L130-L134

Reply:
The Mobile Width story looks a bit small on a Pixel viewport, which I think is caused by the fixed width on the Box here.

<img width="746" alt="Mobile Width flipbook story in the storybook, with Pixel selected from the viewport dropdown." src="https://user-images.githubusercontent.com/59547/204554629-912c262c-2e74-4008-98bf-8d9faf266ad9.png">

Reply:
Can you clarify what you mean by looks a bit small? Are you referring to the size of the icons and thumbnails in the flipbook controls?

Reply:
I think the confusion here is that I named the story "mobile width" when in fact it's a story about the size of flipbook controls' container `div` width - regardless of the device a volunteer is using. I'll rename this story to make it clear that it's showing how container `div` width affects the controls styling.